packages/router/vitest.setup.mts

```diff
@@ -0,0 +1,15 @@
+import '@testing-library/jest-dom/vitest'
+
+import { afterEach, vi } from 'vitest'
+import { cleanup } from '@testing-library/react'
+
+// @ts-expect-error
+globalThis.scrollTo = vi.fn()
```

Comment:
@Tobbe I suspect you'll not like this. I haven't tried to type a mock to scrollTo before and whatever I try fails to make TS happy.

Reply:
Fixed in [91e85c1](https://github.com/redwoodjs/redwood/pull/10674/commits/91e85c1e84e9fd566d549677c196cd636f522f2f)